### PR TITLE
KFSPTS-24217 backport FINP-8250

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

File: src/main/java/org/kuali/kfs/kns/web/struts/action/KualiDocumentActionBase.java [hunk 1]
```diff
@@ -0,0 +1,2514 @@
...
+/**
+ * 
+ * CU customization: backport FINP-8250.
+ * 
+ * Changes from FINP-8250 were applied in base financials to KualiAction.java in
+ * the 2/22/2022 release. KualiAction.java has been merged with
+ * KualiDocumentActionBase.java in the 1/5/2022 release. The fix has been
+ * backported to a copy of KualiDocumentActionBase.java from the 1/28/2021
+ * version of financials. This will need to be updated accordingly when we
+ * upgrade to the 1/5/2021 version of financials so that changes are moved to
```
Comment (cah292) [inline src/main/java/org/kuali/kfs/kns/web/struts/action/KualiDocumentActionBase.java:144]: Did you mean to say "1/5/2022" instead?

**Review by cah292 — CHANGES_REQUESTED** [review]
I noticed one minor change that might be needed to one of the comments, but otherwise the changes look good.

File: src/main/java/org/kuali/kfs/kns/web/struts/action/KualiDocumentActionBase.java (hunk 1 above)
Comment (db483) [reply src/main/java/org/kuali/kfs/kns/web/struts/action/KualiDocumentActionBase.java:144]: Yes, thank you for noticing. Updating.

**Review by cah292 — APPROVED** [review]
Backport looks good, and unit tests pass. Merging.

**Pull request merged**